idle-infrastructure/fcm/src/main/kotlin/com/swm/idle/infrastructure/fcm/users/service/CenterManagerVerificationRejectEventService.kt

```
@@ -0,0 +1,53 @@
+class CenterManagerVerificationRejectEventService(
+    private val fcmClient: FcmClient,
+) {
+
+    private val logger = KotlinLogging.logger {}
+
+    fun send(centerManagerVerificationRejectEvent: CenterManagerVerificationRejectEvent) {
+        val message = createMessage(centerManagerVerificationRejectEvent)
+
+        try {
+            fcmClient.send(message)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)